### DISCUSSION TITLE
Ayamiss, Ossirian and his crystals

### DISCUSSION THREAD

**Discussion by Rholik** [chromiecraft/chromiecraft#3995] [Suggestions]
Starting with Ayamiss her larvas get really fast to the altar and makes for blue people impossible kill it before sacrifice, and the fight being a dps check when is a "Catch up Raid". One shooting the caster who has her agroo and eventually wipe the raid.
Ossirian has a problem and is his concept, with the aura Strength of Ossirian https://classic.wowhead.com/spell=25176/strength-of-ossirian. he is already a overturned boss, now when you overtune that is impossible to tanks get him because one-shot, winds and impossible to taunt. And the crystals had to be way more bigger in the hitbox, you have to put ossirian just above the crystal and the crystal have 3-5 second animation to do the effect, so is impossible to get him quiet without someone die or directly wipe. And remember that's suppose to be a Catch up raid.

**Comment by ohsiros** [comment]
You're not supposed to tank Ossirian when his aura is on, this is only relevant before popping the first crystal to debuff him. Kite him to a crystal to remove it and he won't one-shot. 
Agree that crystal's have a bit wonky hitbox and might need a hot-fix/buff to widen it's area. Second animation is just timing and will be way easier if crystal hitbox is bigger.
